### PR TITLE
fix(loader): restore animation

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #10182

## Summary
While the new lint rule catches non-kebab-cased names at `@function` definition, it does not catch wrong casing when applied. This fixes missed applications.

### CODE REVIEW

File: packages/calcite-components/src/components/loader/loader.e2e.ts [hunk 1]
```diff
@@ -55,4 +55,43 @@ describe("calcite-loader", () => {
...
     expect(loader.getAttribute("id").length).toEqual(36);
   });
+
+  it("animates loader segment offset", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-loader></calcite-loader>`);
+
+    const duration = await page.evaluate(() => {
+      const loader = document.querySelector("calcite-loader");
+      const style = getComputedStyle(loader);
```
Comment (driskull) [inline packages/calcite-components/src/components/loader/loader.e2e.ts:65]: I think you should be able to do: `const style = getComputedStyle(loader).animationDuration;`

File: packages/calcite-components/src/components/loader/loader.e2e.ts [hunk 2]
```diff
@@ -55,4 +55,43 @@ describe("calcite-loader", () => {
...
+      const durationValue = style.getPropertyValue("animation-duration");
+      const match = durationValue.match(/(\d+(\.\d+)?)/);
```
Comment (driskull) [inline packages/calcite-components/src/components/loader/loader.e2e.ts:67]: I don't think you need to do a match here.

File: packages/calcite-components/src/components/loader/loader.e2e.ts (hunk 1 above)
Comment (driskull) [reply packages/calcite-components/src/components/loader/loader.e2e.ts:65]: Applies to other places.

File: packages/calcite-components/src/components/loader/loader.e2e.ts [hunk 3]
```diff
@@ -55,4 +55,43 @@ describe("calcite-loader", () => {
...
+      return parseFloat(match[0]);
+    });
+    console.log("duration", duration);
+
+    const timeAt60Percent = duration * 0.6;
+    console.log("timeAt60Percent", timeAt60Percent);
+
+    const initialOffset = await page.evaluate(() => {
```
Comment (driskull) [inline packages/calcite-components/src/components/loader/loader.e2e.ts:75]: maybe make this a `getStrokeDashOffset` function since its logic is repeated.

**Review by driskull — APPROVED** [review]
👍

**Review by jcfranco — APPROVED** [review]

**Pull request merged**